FinSearchUnified/Helper/StaticHelper.php

```diff
@@ -341,4 +341,20 @@ public static function setQueryInfoMessage(QueryInfoMessage $queryInfoMessage)
             ]
         );
     }
+
+    /**
+     * @param $version
+     *
+     * @return bool
+     */
+    public static function isVersionLowerThan($version)
+    {
+        $shopwareVersion = Shopware()->Config()->get('version');
+
+        if ($shopwareVersion === '___VERSION___') {
```

Comment:
Add a comment on when this applies. Someone reading the code in the future may be confused about that.